frontend/src/libs/components/author/author.tsx

```diff
@@ -0,0 +1,48 @@
+import { type FC } from 'react';
+
+import { Avatar } from '../avatar/avatar.jsx';
```

Comment:
Perhaps it should be imported from `~/libs/components/components.js` because it is exported from there.